.guardrails/ignore

```diff
@@ -1,2 +1,3 @@
 Integrations/Active_Directory_Query/key.pem
-poetry.lock
\ No newline at end of file
+poetry.lock
+Packs/Whois/Integrations/Whois/Whois_test.py
```

Comment:
What is this?

Reply:
The Guardrails job was because of the `pickle`ing of test data in https://github.com/demisto/content/blob/CIAC-5844-whois-add-metrics/Packs/Whois/Integrations/Whois/Whois_test.py#L332-L333, see https://docs.guardrails.io/docs/vulnerabilities/python/insecure_processing_of_data. 

Since we know what we're un`pickle`ing, not sure this is actually a threat. WDYT?

Reply:
Instead of this file use `// guardrails-disable-line` on the line

Reply:
https://github.com/demisto/content/pull/27370/commits/db03805d9abaed746d9a387fbabe01d3c1c6c85e